examples/todo/src/store.ts

```diff
@@ -1,17 +1,17 @@
 import { proxy, useSnapshot } from 'valtio'
 
 export interface Todo {
-  id: number
-  name: string
-  completed?: boolean
+  id: number;
```

Comment:
I think this was what prettier did. How did no-semi give you an error??

Reply:
When I started the todo demo according to README.md, I found an error 'Line 5:3: Parsing error: Unexpected token', so I suggested a fix. But after I migrated this demo with the latest create-react-app, I did not have this problem.

Reply:
Interesting. Let's revisit if we find any env that will cause issues. I guess there will be some workarounds.